packages/craco/README.md

```diff
@@ -552,6 +552,47 @@ const myNewWebpackLoader = {
 addAfterLoaders(webpackConfig, loaderByName("eslint-loader"), myNewWebpackLoader);
 ```
 
+#### throwInvalidConfigError
+
+Throw an error if the webpack configuration changes and does not match your expectations. (For example, `getLoader` cannot find the loader, and `isFound` is `false`.) `create-react-app` or `webpack` might change the structure of the configuration file, so it is important to write defensive code, especially if you are writing a plugin.
+
+Raises an error and crashes Node.js:
+
+```bash
+$ yarn start
+yarn run v1.12.3
+$ craco start
+/path/to/your/app/craco.config.js:23
+            throw new Error(
+            ^
+
+Error: Can't find eslint-loader in the webpack config!
+
+Did you update create-react-app or craco recently?
+Please take a look at some recent issues in craco and create-react-app to see if someone has found a solution:
+
+* https://github.com/sharegate/craco/issues?q=is%3Aissue+webpack+eslint-loader
+* https://github.com/facebook/create-react-app/issues?q=is%3Aissue+webpack+eslint-loader
```

Comment:
Most plugins should be in a standalone repository, there should be a way for the plugin author to add a link to is own repository issues

Reply:
Good point! I actually did do this in the `craco-less` plugin: https://github.com/ndbroadbent/craco-less/blob/master/lib/craco-less.js#L4-L19

It would be great if I could have imported this error helper from the `craco` utility. So I will update this.

Reply:
Hi @patricklafrance, I've just updated the error message helper to include support for a GitHub repo URL, and the NPM package name.

The error message was starting to get quite complicated, so I've also decided to write some tests. You can run these with `yarn test`. I've added `jest` and included a `.travis.yml`, so it should also be easy to get this running on Travis CI. 

That made me think that it would be really helpful to add a few more tests, to make sure everything is working with `react-scripts`. This would make it much easier to update in the future, and you would get instant feedback about any breaking changes.

Reply:
@ndbroadbent I agree with you, @yohanb planned to work on integration tests, not sure about the progress thought?

Reply:
Was thinking of using Azure Pipelines with the free offering with GitHub since it's what I know best.

Reply:
@ndbroadbent @yohanb I created #21 and #22

Reply:
Awesome! Sure Azure Pipelines sounds great as well!